data-products/src/update_pocket_score_data_flow.py

```diff
@@ -0,0 +1,306 @@
+            params=delete_args(one_dict),  # type: ignore
+            sqlalchemy_credentials=unmapped(mysql_creds),  # type: ignore
+        )
+        logger.info(
+            "V1 inserts: %s, updates: %s, deletes: %s", inserts, updates, deletes
+        )
+    else:
+        logger.info("No publisher records to process for the V1 process")
+
+    if len(two_dict) > 0:
+        logger.info("V2 dictionary length %s", len(two_dict))
+        inserts = sqlalchemy_execute.map(
+            statement=unmapped(INSERT_TO_CONTENT_V2_SQL),  # type: ignore
+            params=insert_args(two_dict),  # type: ignore
+            sqlalchemy_credentials=unmapped(mysql_creds),  # type: ignore
+        )
+        updates = sqlalchemy_execute.map(
+            statement=unmapped(UPDATE_CONTENT_V2_SQL),  # type: ignore
+            params=update_args(two_dict),  # type: ignore
+            sqlalchemy_credentials=unmapped(mysql_creds),  # type: ignore
+        )
+        deletes = sqlalchemy_execute.map(
+            statement=unmapped(DELETE_PROCESS_QUEUE_SQL),  # type: ignore
+            params=delete_args(two_dict),  # type: ignore
+            sqlalchemy_credentials=unmapped(mysql_creds),  # type: ignore
+        )
+        logger.info(
+            "V2 inserts: %s, updates: %s, deletes: %s", inserts, updates, deletes
+        )
+    else:
+        logger.info("No publisher records to process for the V2 process")
+
+
+@flow()
+async def update_pocket_score_data():
+    mysql_creds = MozSqlalchemyCredentials()
+
+    # move articles to the queue to be processed from their original holding queue
+    move_to_beta_queue_task = await sqlalchemy_execute(
+        statement=MOVE_TO_PROCESSING_QUEUE_SQL, sqlalchemy_credentials=mysql_creds
+    )
+
+    # Drop all articles from the holding queue since we moved them to our processing queue.  # noqa: E501
+    truncate_content_quality_task = await sqlalchemy_execute(
+        statement=TRUNCATE_CONTENT_QUALITY_SQL,
+        sqlalchemy_credentials=mysql_creds,
+        wait_for=[move_to_beta_queue_task],  # type: ignore
+    )
+
+    # Take the articles from our processing queue and get the data
+    extract_result = await sqlalchemy_query(
+        query=DATA_FOR_SCORE_CALCULATION_SQL,
+        sqlalchemy_credentials=mysql_creds,
+        wait_for=[truncate_content_quality_task],  # type: ignore
+    )
+    # Transform the articles we grabbed to have our new data
+    transform_result = transform(extract_result)
+
+    # Load the processed data back into MySQL with a content score!
+    await load(rows=transform_result, mysql_creds=mysql_creds)
+
+
+FLOW_SPEC = FlowSpec(
+    flow=update_pocket_score_data,
+    docker_env="base",
+    deployments=[
+        FlowDeployment(
+            name="deployment",
+            cron="*/60 * * * *",
+            job_variables={
+                "cpu": 2048,
+                "memory": 4096,
+            },
+        ),
+    ],
+)
```

Comment:
Do we know what this flow is used for? Or will we just make a reasonable effort to keep it running, even if it might not be used anymore?

(To be clear, I have no idea. :grimacing: )

Reply:
yeah at this point, just going to keep it running...it does seem to be referenced in Web